portal-web/test/functional/com/liferay/portalweb/macros/CompanyLogging.macro

```diff
@@ -0,0 +1,89 @@
+definition {
+
+	macro assertCompanyLogIsolated {
+		if ("${portalInstanceName}" == "liferay.com") {
+			var baseCompanyId = JSONCompany.getCompanyId();
+		}
+		else {
+			var baseCompanyId = JSONCompany.getCompanyId(portalInstanceName = "${portalInstanceName}");
+		}
+
+		var logContent = CompanyLogging.downloadCompanyLog(portalInstanceName = "${portalInstanceName}");
+
+		if (!(contains("${logContent}", "companyId=${baseCompanyId}"))) {
+			fail("The ${portalInstanceName} company log file content does not match the expected log content.");
+		}
+
+		var companies = JSONCompany.getAllCompanyNames();
+
+		for (var companyTitle : list "${companies}") {
```

Comment:
SF, in Liferay for loops, the control variable companyTitle should be the singular verbal form of the list.
So it should be of a format like 
```
for (var companyTitle : list "${companyTitles}") {
}
```
or company companies